capstone/src/main/java/com/google/sps/data/Map.java

```diff
@@ -0,0 +1,39 @@
+public class Map {
```

Comment:
LGTM, one more tip would be to make the class name more specific since we already have map objects from the Map API and it may be confusing